src/index.scss

```diff
@@ -7,3 +7,52 @@ html {
 *::after {
   box-sizing: inherit;
 }
+
+.wrapper {
+  padding: 0.5rem;
+}
+
+.email-client {
+  border: 1px solid #ccc;
```

Comment:
Maybe we should put things like colors into Sass variables just to exemplify how those work?